packages/ifpack2/test/unit_tests/Ifpack2_UnitTestBlockRelaxationPerf.cpp

```diff
@@ -106,33 +102,95 @@ RCP<const Epetra_CrsMatrix> create_testmat_e(const Epetra_Map& rowmap)
 
 TEUCHOS_UNIT_TEST(Ifpack2BlockRelaxation, Performance)
 {
+  constexpr bool debug = true;
+  auto myOutPtr = debug ?
+    Teuchos::getFancyOStream (Teuchos::rcpFromRef (std::cerr)) :
+    Teuchos::rcpFromRef (out);
+  auto& myOut = *myOutPtr;
+
+  myOut << "Ifpack2 BlockRelaxation performance comparison: Epetra vs. Tpetra"
+        << std::endl;
+
```

Comment:
Why create a new ostream instead of using `out`?

Reply:
@jhux2 I was trying to debug the test; this led to discovery of #3192.  The test was bombing out before it could print any useful information, so I replaced `out` with a `FancyOStream` that wraps `std::cerr`.  I think I've tried just overwriting `out` before, but it didn't work.